routes/create.js

```diff
@@ -0,0 +1,86 @@
+import express from 'express';
+import multer from 'multer';
+import fs from 'fs';
+import async from 'async';
+import fileHandler from '../src/file-handler';
+import InvertedIndex from '../src/inverted-index';
+import validateUpload from '../src/validate-upload';
+const upload = multer({ dest: 'uploads' });
+const router = express.Router();
+const invertedIndex = new InvertedIndex();
+
+router.post('/', upload.array('files'), (req, res) => {
+  const indices = {};
+
+/*
+  const files = req.files;
+  // Check that files were uploaded
+  if (files === undefined || files.length === 0) {
+    return res.send({ error: 'no file uploaded' });
+  }
+  let count = 0;  // Keep count of number of indexed files
+  files.forEach((file) => {
+    const originalFileName = file.originalname;
+    const filePath = file.path;
+
+    // Check for valid file type
+    const fileExtension = (originalFileName.split('.').pop()).toUpperCase();
+    if (fileExtension !== 'JSON') {
+      count += 1;
```

Comment:
How about I create a separate branch that doesn't have 'superfluous' comments while retaining this branch with comments? I figured some parts of my code might require a lot of explanations, that's why I wrote such large amounts of comments.